bears/general/GenericHelpers.py

```diff
@@ -0,0 +1,162 @@
+from coala_utils.string_processing.Core import unescaped_search_for
```

Comment:
while it's fine to make these methods available, I don't really like the file name "GenericHelpers". It's not neccesarily where people would go looking if they need the functionality you provide here. Also it *might* be a good idea to move the error definition somewhere else, but we don't have any other custom errors/ exceptions so far, do we?